sale_product_pack/models/sale_order_line.py

```diff
@@ -84,8 +84,7 @@ def create(self, vals_list):
                 line.expand_pack_line()
                 res |= line
             else:
-                new_vals.append(elem)
-        res |= super().create(new_vals)
+                res |= super().create([elem])
```

Comment:
This is incorrect, as it's serializing the creation. If you want to keep order, play with other things like sequence, but I don't understand why they aren't properly ordered already. Please include steps to reproduce the problem.

Reply:
Hi! I added the steps to reproduce error and a video on description

Reply:
The video is not shown, but as said, the solution for the problem shouldn't be to serialize the creation.

Reply:
Hi @pedrobaeza I made some changes!